k8s/ingress.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: flow-ingress
+  namespace: flow
+  annotations:
+    kubernetes.io/ingress.class: alb
+    alb.ingress.kubernetes.io/scheme: internet-facing
+    alb.ingress.kubernetes.io/target-type: ip
+    alb.ingress.kubernetes.io/ssl-redirect: '443'
+    alb.ingress.kubernetes.io/certificate-arn: arn:aws:acm:us-east-1:132880019009:certificate/YOUR_CERT_ARN
```

Comment:
This wouldn't work would it? If we're going to use yaml manifests to deploy kubernetes resource we would probably need to make this configurable, probably by making these manifests part of a helm chart.